example.env

```diff
@@ -8,3 +8,8 @@ PR_CLIENT_SECRET=replaceme
 REACT_APP_DISABLE_CODE_3=
 REACT_APP_DISABLE_PILOT_HOSPITALS=
 SESSION_SECRET=makeitasecretinprod
+
+EMS_USER=op.ems@c4sf.me
+EMS_PASS=abcd1234
```

Comment:
Do we want these passwords in the repo?

Reply:
I think these dummy seed accounts/passwords are fine, they're obviously placeholder...